CHANGELOG.md

```diff
@@ -2,6 +2,39 @@
 
 All notable changes are documented in this file. Release numbers follow [Semantic Versioning](http://semver.org)
 
+## 0.10.0.CR1
+September 10th, 2019 [Detailed release notes](https://issues.jboss.org/secure/ReleaseNote.jspa?projectId=12317320&version=12342542)
+
+### New features since 0.10.0.Beta3
```

Comment:
Shouldn't all the sections be `since 0.10.0.Beta4`?

Reply:
@renatomefi Spot on, thanks for reporting!